### ISSUE TITLE
Cant install Jackett on High Sierra

### ISSUE THREAD

**Issue by Rival-Dev** [Jackett/Jackett#2148]
every time I try to install jackett I get this error

```
Last login: Tue Nov 14 19:10:32 on ttys000
/Users/nik/Downloads/Jackett\ 2/install_service_macos ; exit;
Niks-Mac-Pro:~ nik$ /Users/nik/Downloads/Jackett\ 2/install_service_macos ; exit;
/Users/nik/Downloads/Jackett 2/install_service_macos: line 32: /Users/nik/Library/LaunchAgents/org.user.Jackett.plist: No such file or directory
/Users/nik/Library/LaunchAgents/org.user.Jackett.plist: No such file or directory
ERROR: Could not launch agent. The installation might have failed.
Please open an issue on https://github.com/Jackett/Jackett/issues and paste following information:
Mono directory: `/Library/Frameworks/Mono.framework/Versions/Current/Commands`
Jackett directory: `/Users/nik/Downloads/Jackett 2`

logout
Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.

[Process completed]
```

Jackett version:  0.8.402
Mono version (if not using Windows):  5.4.1.6

-------------------------------

**Comment by kaso17** [comment]
Please try running a `mkdir -p ~/Library/LaunchAgents/` before running the install script.

**Comment by kaso17** [comment]
@Rival-Dev did this fix your problem?

**Comment by Rival-Dev** [comment]
Yes

**Comment by kaso17** [comment]
fixed via fc3a1e21d3fe4aa307a94257801d8a14e6942421
